### PR TITLE
feat: add --no-integrations option

### PR DESCRIPTION
**Pull request by maxandersen**
realized #1776 didn't require much for the first MVP.

adds a `--no-integrations` option to disable execution of integrations.

### CODE REVIEW

**Comment by maxandersen** [comment]
opening as draft as not sure this is how it should look. 

@jmini give it a try for your executable and tell me if works for you or things missing.

File: src/main/java/dev/jbang/util/Util.java [hunk 1]
```diff
@@ -77,7 +77,7 @@ public class Util {
 			Pattern.MULTILINE);
 
 	public static final Pattern mainClassMethod = Pattern.compile(
-			"(?<=\\n|\\A)(?:public\\s)\\s*(class)\\s*([^\\n\\s]*)");
```
Comment (quintesse) [inline src/main/java/dev/jbang/util/Util.java:80]: This is an unrelated change, isn't it?

**Comment by quintesse** [comment]
@maxandersen I updated the PR with the "actual" code. (The `--(no-)integrations` option changes compile output, like for example `--native`, and should therefore be part of the `Project` options. I changed the PR to reflect that.

(I _am_ starting to dislike having to copy all options to the `Alias` class though)

File: src/main/java/dev/jbang/util/Util.java (hunk 1 above)
Comment (maxandersen) [reply src/main/java/dev/jbang/util/Util.java:80]: yeah thatwas definitely not intended.

**Comment by quintesse** [comment]
Hmmm weird, the error is still there...

**Comment by quintesse** [comment]
Ok, problem fixed and I added support for a `//NOINTEGRATIONS` tag as well.
